openedx_tagging/core/tagging/import_export/actions.py

```diff
@@ -72,7 +72,9 @@ def _get_tag(self) -> Tag:
         """
         Returns the respective tag of this actions
         """
-        return self.taxonomy.tag_set.get(external_id=self.tag.id)
+        if self.tag.id:
+            return self.taxonomy.tag_set.get(external_id=self.tag.id)
```

Comment:
```suggestion
            try:
                return self.taxonomy.tag_set.get(external_id=self.tag.id)
            except Tag.DoesNotExist:
                return self.taxonomy.tag_set.get(value=self.tag.value, external_id=None)
```
What about if the tags were imported without an external ID, but then external IDs were added to the data and the import happens again? I think something like this ^ would handle that situation nicely.

Reply:
Yes you are right, I updated this [here](https://github.com/openedx/openedx-learning/pull/135/files#diff-caeae510c03c9354f48c9cb75e95fc0fb5322180c1b874622c237a461091b6f7R75)